config/schedule.rb

```diff
@@ -163,6 +163,13 @@
     at: '11:00am',
     interruptable: true,
   },
+  {
+    task: 'driver:hmis_external_apis:import:ac_projects',
+    frequency: 1.day,
+    at: '8:00am', # FIXME: when are we expecting the file? I'll set this to a few hours later
+    interruptable: true,
+    trigger: ENV['IS_AC'] == 'true' # FIXME: will this be automatic on staging? How to best detect this installation?
```

Comment:
I think this has to be an environment variable, unless we can find an easy way to look in the db.  Maybe `ENV['REMOTE_PROJECTS']` or something that has to do with the import process?  Also, maybe it makes sense to just add this to `eto:import:demographics_and_touch_points` which runs daily.  I'm thinking we should probably collapse the two together and call it something like `fetch_remote_data`....

Reply:
Let me know which way you want to go. Talking to the database will be more complicated and take (much?) longer. It's definitely possible.